legend-depot-store-metrics/src/main/java/org/finos/legend/depot/store/metrics/MetricsModule.java

```diff
@@ -20,21 +20,26 @@
 import com.google.inject.Singleton;
 import com.google.inject.name.Named;
 import org.finos.legend.depot.schedules.services.SchedulesFactory;
+import org.finos.legend.depot.store.metrics.api.QueryMetricsRegistry;
 import org.finos.legend.depot.store.metrics.services.QueryMetricsHandler;
+import org.finos.legend.depot.store.metrics.services.InMemoryQueryMetricsRegistry;
 
 public class MetricsModule extends PrivateModule
 {
     @Override
     protected void configure()
     {
         bind(QueryMetricsHandler.class);
+        bind(QueryMetricsRegistry.class).to(InMemoryQueryMetricsRegistry.class);
```

Comment:
you dont need this... move the intializtion from services to this module ...u dont need this bind